Subsurface/Node/Common/Peripherals/inc/PWMInterface.h

```diff
@@ -0,0 +1,45 @@
+#include "stm32f3xx_hal.h"
+#include <cstdint>
+
+#ifndef PWMINTERFACE_H
+#define PWMINTERFACE_H
+
+class PWMInterface
+{
+public:
+    // Constructor
+    PWMInterface(uint64_t clockSpeed, uint32_t preScaler, uint32_t period,
+    uint32_t pulse, TIM_HandleTypeDef timer, uint32_t channel);
```

Comment:
See my comments on Andrew's I2C PR about how to format these multi-line declarations, nbd for here but just for reference going forward